app/components/HeaderNotifications/HeaderNotifications.css

```
@@ -22,10 +22,13 @@ html[data-theme='dark'] .notifications {
 
 .notification {
   border-bottom: 1px solid lightgray;
-  padding: 15px;
+  padding-top: 15px;
+  padding-right: 15px;
+  padding-left: 15px;
```

You can actually rewrite this to
```suggestion
  padding: 15px 15px 0 15px;
```

See https://developer.mozilla.org/en-US/docs/Web/CSS/padding#syntax

---

fixed